05_book_titles/book.rb

```diff
@@ -1,3 +1,14 @@
 class Book
-# write your code here
+    attr_accessor :title
+
+    exceptions = %w[and the a an at to on in of]
+    def title=(new_title)
+      @title = new_title.split.map.with_index do |word, index|
+        if index.zero? || !exceptions.include?(word.downcase)
```

Comment:
Nit: You can avoid one of the checks inside the loop if you just capitalize up front first.

```suggestion
      @title = new_title.capitalize.split.map.with_index do |word, index|
        if !exceptions.include?(word.downcase)
```